guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -10,13 +10,22 @@ This information allows Katello to display manifest labels, annotations, and inf
 As a result, migrations must be performed to pull this content from manifests into the database.
 
 ifdef::katello[]
-This migration takes time, so if you depend on container content and need minimal upgrade downtime, run the following command on {ProjectServer} for a pre-migration:
-
+.Procedure
+. This migration takes time, so if you depend on container content and need minimal upgrade downtime, run the following commands on {ProjectServer} for a pre-migration:
++
 [options="nowrap" subs="attributes"]
 ----
 # foreman-maintain advanced procedure run pulpcore-container-handle-image-metadata
 ----
 This command migrates data in the background without any need for downtime and reduces future upgrade downtime.
```

Comment:
General question: the command runs in the foreground? If it can take a long time, should we recommend running it in `tmux` like we do upgrades? That way you can detach the process. Or can you rerun it multiple times if it stopped?

Reply:
@ianballou Can you help with this as well?

Reply:
The "in the background" part is a bit misleading I realize -- the original intent was to communicate that it can run while Foreman is still operating normally. 
If the process stops, you should be able to run the command over and over again, but I think it's worth saying to run it in a tmux window.